geoportailv3/templates/index.html

```diff
@@ -46,7 +46,12 @@
           </div>
         </div>
 
-        <div class="offline-msg alert-danger" translate>You are currently offline.</div>
+        <div class="offline-msg alert-danger" translate>
+          <span ng-if="!mainCtrl.offlineMode.isEnabled()">You are currently disconnect.
+            <a ng-click="mainCtrl.offlineMode.activateOfflineMode()" translate>Enable offline mode</a>
```

Comment:
The link should only be visible if we are not in offlineMode.

Reply:
The test is done on the `span`

Reply:
Said differently:
- when network is disconnected: "You are currently disconnected" should be visible;
- when not offline: "Enable offline mode" should be visible.

Your code hides the network status when offline is enabled.